

<launch>

  <node     	
	  name    = "lidar_testing"	
  	pkg     = "tree_identification"
  	type    = "lidar_testing"
  	output  = "screen"
	>
    	<param 
			name = "bag_name"
			value = "/home/conor/SERDP/data/vandenberg/input/lidar.bag" /> <!--"/home/conor/ros_data/trees/Aerial Lidar/Subset Scenes/Hornsby/b3.bag" /-->
		<param 
			name = "bag_topic"
			value = "lidar" />
	    <param 
	      name = "leaf_size"
	      value = "0.05" />
	    <param 
	      name = "veg_intensity_threshold"
	      value = "50" />
	    <param 
	      name = "veg_normal_threshold"
	      value = "0.1" />
	    <param 
	      name = "dem_pixel_size"
	      value = "1.0" />
	    <param 
	      name = "dem_pixel_size_coarse"
	      value = "15" />
	    <param 
	      name = "cluster_tolerance"
	      value = "1.2" />
	    <param 
	      name = "reattachment_search_radius"
	      value = "3.0" />
	    <param 
	      name = "mls_radius"
	      value = "3.0" />
	    <param 
	      name = "image_scale_min_height"
	      value = "125" />
	    <param 
	      name = "image_scale_max_height"
	      value = "145" />
	    <param 
	      name = "gap_filling_radius"
	      value = "2.5" />
	    <param 
	      name = "mls_radius"
	      value = "3.0" />
	</node>

	<rosparam  command="load"  file="$(find tree_identification)/param/hornsby.yaml"/>

</launch>